src/stats.jl

```diff
@@ -191,30 +191,39 @@ function hpd(chn::Chains; alpha::Real=0.05, kwargs...)
 end
 
 """
-    quantile(chains[; q = [0.025, 0.25, 0.5, 0.75, 0.975], append_chains = true, kwargs...])
+    quantile(chains, q = [0.025, 0.25, 0.5, 0.75, 0.975]; alpha, beta, kwargs...)
+    quantile(chains; q, alpha, beta, kwargs...)
 
 Compute the quantiles for each parameter in the chain.
 
 Setting `append_chains=false` will return a vector of dataframes containing the quantiles
 for each chain.
+
+# Arguments
+
+- `chains`: A `Chains` object.
+- `q`: A vector of quantiles.
+- `alpha`, `beta`: See `?quantile` for details.
+- `kwargs...`: A set of keyword arguments to be passed to `summarize`.
 """
 function quantile(
     chains::Chains;
-    q::AbstractVector = [0.025, 0.25, 0.5, 0.75, 0.975],
-    append_chains = true,
+    q::AbstractVector=[0.025, 0.25, 0.5, 0.75, 0.975],
+    alpha::Real=0,
+    beta::Real=alpha,
     kwargs...
 )
     # compute quantiles
     funs = Function[]
     func_names = @. Symbol(100 * q, :%)
-    for i in q
-        push!(funs, x -> quantile(cskip(x), i))
+
+    for prob in q
+        push!(funs, x -> quantile(cskip(x), chains.weights, prob; alpha=alpha, beta=beta))
```

Comment:
This is not correct: `cskip` skips missing entries and hence the length of `cskip(x)` and `chains.weights` might not match. `alpha` and `beta` could be moved to a separate PR.